standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+### 4.2. Managing Fetched Bundles
+Clients of the bundle endpoint MUST store the latest SPIFFE bundle each time it is retrieved. The sequence number field of the trust bundle SHOULD be used when comparing the freshness or ordering of two trust bundles. If the trust bundle omits the sequence number, operators SHOULD consider the most recently retrieved bundle to be up-to-date.
+
+Operators MAY locally update the SPIFFE bundle of a foreign trust domain at any time. In this case, the locally updated version of the bundle is considered the latest until replaced by a subsequent refresh.
+
+Bundle contents from different trust domains MUST NOT be merged into a single, larger bundle. Doing so would enable one trust domain to forge identities belonging to another trust domain in the eyes of the validator with the unified bundle. As such, it is very important to ensure that bundles received from foreign trust domains be kept distinct, and clearly reflect the trust domain name to which they belong. Please see the [Security Considerations](#7-security-considerations) section for more information.
+
+### 4.3. Endpoint Address Stability
+Once foreign trust domains begin relying on a specific endpoint URL, it is a delicate and error-prone procedure to migrate all clients of the endpoint to a replacement endpoint URL. Therefore, the safest course of action is to prefer a stable endpoint URL.
+
+## 5. Serving and Consuming a SPIFFE Bundle Endpoint
+This specification defines two supported profiles for SPIFFE bundle endpoint servers, both of which are based on HTTPS. One relies on the use of Web PKI to authenticate the endpoint and the other leverages SPIFFE authentication. SPIFFE bundle endpoint clients MUST support both of these profiles while SPIFFE bundle endpoint servers MUST support at least one.
+
+Bundle endpoint servers that support TLS-based profiles (e.g. `https_web` or `https_spiffe`) MUST adhere to the [Mozilla intermediate compatibility][5] requirements unless otherwise stated by the profile in use.
+
+### 5.1. Endpoint Parameters
+Prior to retrieving a bundle from a SPIFFE bundle endpoint, clients MUST be configured with the following three parameters: (1) the URL of the SPIFFE bundle endpoint, (2) the endpoint profile type, and (3) the trust domain name to associate with the bundle endpoint. The first two parameters indicate the location of the bundle endpoint and how to authenticate it. Since trust bundles do not contain a trust domain name, clients use the third parameter to associate a downloaded bundle with a specific trust domain name. Specific [endpoint profiles](#52-endpoint-profiles) (such as `https_spiffe`, as described below) MAY define additional mandatory configuration parameters.
```

Comment:
I feel like we should say something about what clients are supposed to do when a trust bundle contains unrecognized parameters:

Clients SHOULD ignore configuration parameters that are not defined in the endpoint profiles below.

Reply:
These are client config parameters, not parameters in the trust bundle.

I don't think we can reasonably (or should) say that implementers can't add additional parameters.  As a trivial example, I think it would be reasonable for an implementer to provide a config parameter that sets or overrides the refresh interval for a specific endpoint.